### PR TITLE
Update GH Version to 0.9

### PR DESCRIPTION
**Pull request by rap2363**
@karussell This PR updates the traffic demo code so that it is compatible with GH version 0.9.0. The biggest change is in having the `DataUpdater` instantiated with an injected GraphHopper dependency instead of passing it in as a constructor arg. Let me know if you have questions!

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20170516</version>
+        </dependency>
```
Comment (karussell) [inline pom.xml]: This shouldn't be necessary anymore as we use jackson

**Comment by karussell** [comment]
Cool, thanks!

File: src/main/java/com/graphhopper/traffic/demo/DataUpdater.java [hunk 2]
```diff
@@ -27,15 +28,16 @@
  */
 public class DataUpdater {
 
+    @Inject
+    private GraphHopper hopper;
```
Comment (karussell) [inline src/main/java/com/graphhopper/traffic/demo/DataUpdater.java]: Looks good this way!

File: pom.xml (hunk 1 above)
Comment (rap2363) [reply pom.xml]: Ok, I'll need to change some stuff that relies on this, but np.

**Comment by karussell** [comment]
Thanks again!

**Pull request merged**